charts/kvisor/values.yaml

```diff
@@ -156,17 +156,13 @@ controller:
 #    capabilities:
 #      drop: [ ALL ]
 
-  resources: { }
-    # We usually recommend not to specify default resources and to leave this as a conscious
-    # choice for the user. This also increases chances charts run on environments with little
-    # resources, such as Minikube. If you do want to specify resources, uncomment the following
-    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-    # limits:
-    #   cpu: 100m
-    #   memory: 128Mi
-    # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+  resources:
+    requests:
+      cpu: 100m
+      memory: 256Mi
+    limits:
+      cpu: 200m
+      memory: 512Mi
```

Comment:
For memory it should be request = limit IMHO

Reply:
why do you think so? I am not against it, just want to hear the reasoning.

Reply:
CPU is a "fluid" resource, meaning that if there is pressure on the node, the pod will simply get less. This is not possible for memory. Meaning, if the node comes into a memory pressure situation, it will kill the pod to make room for resources (I do not recall the exact details, as there are some steps involved to find the best pod to kill). If you have memory limit = resource, your pod will run in the Guaranteed QoS class, meaning it should not get killed. 

Especially in combination with CAST, our nodes will be bin-packed to the max.

Reply:
I undersand. Pushed changes.